reem.CSS

```diff
@@ -1,8 +1,16 @@
+@media screen and (min-width: 640px){
+    .flex-container {
+        display: flex;
+    }    
+}
```

Comment:
Can't see any  elements with this class yet? Wondering if you meant to add this class in this PR?

We could add the flex-box in a separate bit of work later if not?

Reply:
![4](https://user-images.githubusercontent.com/73262473/99744518-f4779d80-2b2b-11eb-9748-8739a14593e7.PNG)
i tried to fix it with flexbox but i cant
what is the PR mean??

Reply:
For now let's delete the media query and flex box selector. Let's come back to the flex box work later so we don't get overwhelmed.

Reply:
The work in this PR is focused on making the layout consistent across pages e.g. same header and footer, styling etc.

Reply:
![Capture5](https://user-images.githubusercontent.com/73262473/100171832-236e8480-2f1b-11eb-8242-c55a37cf6667.PNG)

hello Adam...i tried to fix it just please have  a look for it .... if thats wrong lets keep the flexbox for later.. thank you

Reply:
Oops! Looks like the CSS is broken.

Looks like you need to update the filename `styles.CSS` to `styles.css` so that the HTML file can find it again :wink:.

Let's do the flex-box work later - in a separate PR. Less things to think about and it will help you learn about the scope of PR's too.

Best practice to keep pull requests small and focused so you can easily revert them if there are problems later :smile: